pkg/components/component.go

```diff
@@ -54,20 +54,39 @@ type Component interface {
 	IsUpdatable() bool
 }
 
-type componentBase struct {
+type labellable struct {
```

Comment:
Let's use this as a cause to write a couple of comments explaining the meaning of this struct, and what are Name and Label. It is hard to recall their difference for me.

Reply:
Fun thing is it seems that GetLabel was not used at all. So I deleted it.